src/backend/base/langflow/components/tools/GleanAPI.py

```diff
@@ -0,0 +1,78 @@
+import httpx
+
+from typing import List, Optional
+from urllib.parse import urljoin
+
+from langchain_core.tools import StructuredTool
+
+from langflow.base.langchain_utilities.model import LCToolComponent
+from langflow.inputs import SecretStrInput, StrInput, NestedDictInput, IntInput
+from langflow.field_typing import Tool
+
+from pydantic.v1 import Field, create_model
+
+
+class GleanAPIComponent(LCToolComponent):
+    display_name = "Glean Search API"
+    description = "Call Glean Search API"
+    name = "GleanAPI"
+
+    inputs = [
+        StrInput(
+            name="glean_api_url",
+            display_name="Glean API URL",
+            required=True,
+            value="https://datastax-be.glean.com/rest/api/v1/",
```

Comment:
user should be able to specify this endpoint

Reply:
So right now they can, but i put the datastax one as the default - should i just leave no default you think?

Reply:
I think probably no default is best